samples/tab-calendar/ts/README.md

```diff
@@ -0,0 +1,161 @@
+   - `{{CLIENT_ID}}` - Generated from Step 1 (Application (client) ID)is the application app id
+   - `{{CLIENT_SECRET}}` - Generated from Step 1.14, also referred to as Client secret
+
+5. Open .env file from this path folder `samples/tab-calendar/ts` and update 
+   - `{{REACT_APP_AZURE_APP_REGISTRATION_ID}}` - Generated from Step 1 (Application (client) ID)is the application app id
+   - `{{REACT_APP_BASE_URL}}` - Your application's base url. E.g. https://12345.ngrok-free.app if you are using ngrok.
+     
+6. We have two different solutions to run, so follow below steps:
+ 
+- In a terminal, navigate to `samples/tab-calendar/ts/api-server` folder, Open your local terminal and run the below command to install node modules. You can do the same in Visual studio code terminal by opening the project in Visual studio code
+
+    ```bash
+    npm install
+    ```
+
+    ```bash
+    npm start
+    ```
+- The server will start running on 5000 port
+
+- In a different terminal, navigate to `samples/tab-calendar/ts` folder, Open your local terminal and run the below command to install node modules. You can do the same in Visual studio code terminal by opening the project in Visual studio code 
+
+    ```bash
+    npm install
+    ```
+
+    ```bash
+    npm start
+    ```
+- The client will start running on 3978 port
+
+4. Setup Manifest for Teams
+
+- **This step is specific to Teams.**
+    **Edit** the `manifest.json` contained in the `Manifest/` folder to replace with your MicrosoftAppId (that was created in step1.1 and is the same value of MicrosoftAppId in `.env` file) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
+    **Edit** the `manifest.json` for `validDomains` and replace `{{domain-name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app`.
+    **Zip** up the contents of the `Manifest/` folder to create a `manifest.zip` or `Manifest_Hub/` folder to create a `manifest_Hub.zip`
+    **Upload** the `manifest.zip` to Teams (in the left-bottom *Apps* view, click "Upload a custom app")
+
+## Running the sample
+
+**Install App:**
+
+![InstallApp](Images/Install.png)
+
+**On the side bar, select More Apps. Your sideloaded app title appears among your installed apps:**
+
+![AppOutlook](Images/AppOutlook.png)
+
+**Select your app icon to launch and preview your app running in Outlook on the desktop:**
+
+![HomeScreen](Images/HomeScreen.png)
+
+**Add Details for Compose Meeting:**
+
+![EnterComposeMeeting](Images/EnterComposeMeeting.png)
+
+**Click ComposeMeeting:**
+
+![ClickComposeMeeting](Images/ClickComposeMeeting.png)
+
+**Click View:**
+
+![ClickView](Images/ClickView.png)
+
+## Further Reading.
+[Calendar Module](https://learn.microsoft.com/en-us/javascript/api/@microsoft/teams-js/calendar?view=msteams-client-js-latest)
```

Comment:
Add this link as well:
https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365#calendar

Reply:
Fixed